app/views/assets/_form.html.slim

```diff
@@ -1,4 +1,4 @@
-= form_for @asset, url: {action: "create"} do |f|
+= form_for @asset, :html => { :multipart => true , :class => 'form-horizontal' } do |f|
```

Comment:
Rather don't use => if you don't have to 

So make it html: { multipart:        etc.